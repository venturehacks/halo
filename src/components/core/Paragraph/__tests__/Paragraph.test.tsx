import { render } from 'enzyme';
import React from 'react';

import { Paragraph } from '../index';

test('smoke', () => {
  const component = render(<Paragraph>The story begins...</Paragraph>);
  expect(component.text()).toContain('The story begins...');
});
