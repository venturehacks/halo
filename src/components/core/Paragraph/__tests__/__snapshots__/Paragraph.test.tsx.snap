// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph snapshot contrast 1`] = `
<p
  class="component flow __halo_textContrast_dark_A __halo_fontSizeMap_size--md __halo_lineHeight_default"
>
  single A
</p>
`;

exports[`Paragraph snapshot flow 1`] = `
<p
  class="component __halo_fontSizeMap_size--md __halo_lineHeight_default"
>
  no flow
</p>
`;

exports[`Paragraph snapshot line height override 1`] = `
<p
  class="component flow __halo_fontSizeMap_size--xs __halo_lineHeight_dense"
>
  dense line height override
</p>
`;

exports[`Paragraph snapshot size 1`] = `
<p
  class="component flow __halo_fontSizeMap_size--2xl __halo_lineHeight_default"
>
  size 2xl
</p>
`;
